### DISCUSSION TITLE
Activation Function

### DISCUSSION THREAD

**Discussion by KostasVog20** [lululxvi/deepxde#1648] [Q&A]
Greetings everyone, including @lululxvi!

I'm interested in learning more about activation functions and their functionality within this package. Specifically, I'd like clarification on whether when we specify "tanh" as the activation function, it is applied to both the hidden layers and the output layer, or if the output layer defaults to a "linear approximation" activation function. Furthermore, I'm curious if it's possible to implement the "tanh" activation function for all hidden layers while utilizing a "relu" activation function solely for the output layer without requiring any modifications to the existing code.

**Comment by lululxvi** [comment]
> when we specify "tanh" as the activation function, it is applied to both the hidden layers and the output layer, or if the output layer defaults to a "linear approximation" activation function.

The latter.

> if it's possible to implement the "tanh" activation function for all hidden layers while utilizing a "relu" activation function solely for the output layer without requiring any modifications to the existing code.

You need to implement the code yourself.
